cmd/svcat/svcat_test.go

```diff
@@ -103,6 +103,10 @@ func TestCommandOutput(t *testing.T) {
 		{name: "list all instances", cmd: "get bindings --all-namespaces", golden: "output/get-bindings-all-namespaces.txt"},
 		{name: "get binding", cmd: "get binding ups-binding -n test-ns", golden: "output/get-binding.txt"},
 		{name: "describe binding", cmd: "describe binding ups-binding -n test-ns", golden: "output/describe-binding.txt"},
+
```

Comment:
These validation tests can be moved here once you start using the helper methods mentioned above:

https://github.com/kubernetes-incubator/service-catalog/blob/4dfd13c2b48e5fb33ee048d5c332320bd2502790/cmd/svcat/svcat_test.go#L42

Then we can add a successful / happy path test to this test. You can generate the golden file by doing the following:

1. `touch output/completion.txt`
1. `go test ./cmd/svcat -update`
1. output/completion.txt should now contain the output of the completion command